s3backgrounddelete/s3backgrounddelete/config/s3_background_delete_config.yaml

```diff
@@ -17,6 +17,9 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+version_config:                             # Section for version of S3 background delete config
+   version: 1                                              # Default version for S3 background delete config
```

Comment:
I have always observed that version is always a 3 dot value, like 1.0.0.

Can we rethink on why to stick to single digit version ? @nileshgovande @bkirunge7 !